akka-pi-os-dhcp-32.yml

```diff
@@ -26,7 +28,9 @@ users:
     sudo: ALL=(ALL) NOPASSWD:ALL
     shell: /bin/bash
     groups: users,docker,video
+    ssh-import-id: None
     plain_text_passwd: akkapi
+    ssh-authorized-keys: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDkrgM1almJcqu6FaKDrscUVghlJOO6m7wayVnmH0bQEv4uQQ9SZmkCegrGJ63iMI8X0xUShkwYLippePpj0RiucL1e9/FtzqSdu6IHaPFJg7w5G6cEV2c1mvyJRHpi86+Yt3t4z67aSw77uAWkNFKShtSEbePfF5WX/LGknxRND6f3db+SmmOSy5w5qgC4GmG31gnJX6j7s0xWRiMicyYdGzKOKB4VkQi2YY7bj7hwri4EUHgIZlYuxST57L68CyTVwnF8c39dK6xflea4fkn+3TlNakFw0rrP3iqy8bkaU2cHnMyWIsYF4fbec1JnKngtIGiungvV9QTJzTpMF7ufEFiYl2uzCZgZGhmXsFDo9WoMjKHTJcbM0cm2OWWOrllOGdY2V580szaJR293jQx0OvNhxA3kTAe1aNmpBOfWhhjxxQsPCaPeHTdNG2L3iN9gYT8mms84ENO7OSUYsiMYMas9EMybyYY3Mvd4NUruIxlRgT84kpgyecC/8iQJwvI7vkdZRLlzQ0B614VCA3RH27ZULODSCPz2oFHFfHrMvdxGa4FTYJCWQFBAltWuPjQ8wmNdnKWX85jlc9MPlJ5WK8VIC1f5hVJd6qjSt009WNEt/y6h9xb5qixgZmBRkGpNe/2fyzx1sn0gyVm90wQXDSoT1fWhxZKJciPuHMvYcw== eric.loots@gmail.com
```

Comment:
Does this mean we need to import another piece of keys for a local user to be able to log in? Meaning we don't need to generate key with https://github.com/lightbend/Pi-Akka-Cluster/blob/master/docs/Hypriot-OS-Course-Preparation-Instructions.md#configure-password-less-login ?

Reply:
Yes, this means that you can add your ssh public key to the file which saves you some time by not having to copy and putting it in the .ssh directory of a freshly installed RP!